packages/embed/src/lib/embed-messages.ts

```diff
@@ -55,7 +47,7 @@ export type WindowDimensions = z.infer<typeof WindowDimensionsSchema>;
 export const ClientToFrameActionSchema = z.union([
   z.object({
     type: z.literal('update_config'),
-    payload: SocketConfigSchema,
+    payload: z.custom(() => true), // satisfy the linter for now
```

Comment:
Is this temporary?

Reply:
Yup -- I wanted to get more clarity on this. Wasn't sure why we were were checking the schema here?

Reply:
At Fern, we typically only check schema for things that are stored in a DB, passed over the wire as JSON -- but in this case the `SocketConfigSchema` is used for neither. It's just used to instantiate `HumeClient` and call `chat.connect()`

Reply:
At Hume, we validate that any object that crosses an application boundary. Any object can be sent through `.postMessage` and by checking it when it is received, we can trust that it is the correct type downstream. This helps to surface errors earlier so that user experience isn't impacted.

Here's a great article that discusses this concept: https://www.epicweb.dev/fully-typed-web-apps